src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -193,6 +193,12 @@ protected void LookupMembersInternal(LookupResult result, NamespaceOrTypeSymbol
             }
 
             getCompatibleExtensions(this, type, compatibleExtensions, originalBinder, basesBeingResolved);
+
+            // Sort extensions from more specific to less specific
```

Comment:
It looks like we actually sort the other way around - from less specific to more specific.

Reply:
Indeed. `isLessSpecificExtension` was swapping arguments. Fixed order and added named arguments to clarify.

Reply:
Was this "bug" visible in some tests?

Reply:
Nah, it was a double-negative situation.